.circleci/config.yml

```diff
@@ -1,5 +1,16 @@
 version: 2
 jobs:
+  code style check:
```

Comment:
are spaces allowed in the name of the job?

Reply:
Look like allowed. Just for improve readability in CI report here